utils/average_checkpoints.py

```diff
@@ -115,7 +121,13 @@ def get_parser():
         default="",
         type=str,
         nargs="?",
-        choices=["acc", "bleu", "cer_ctc", "loss", "perplexity"],
+        choices=["acc", "bleu", "cer", "cer_ctc", "loss", "perplexity"],
+    )
+    parser.add_argument(
+        "--max-epoch",
+        default=10000000,
+        type=int,
+        nargs="?",
```

Comment:
How about this option?
How is this option used?

Reply:
@sw005320 I usually use this option to check overfitting and compare multiple models epoch by epoch. If you don't prefer it, I'll remove it. But I'd like to add it in another PR.

Reply:
Thanks!
We can keep this option then.